classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
```

Comment:
- [ ] You have successfully set up the has-many association(a classroom has many students). This was wonderful! :+1: Now let's make sure that the belongs-to(student belongs to a classroom) association is equally set up in the **Student** class by adding a method that serves for that specific. I think the link to this [awesome article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships
) would be of help to you.

Reply:
Well resolved ✔️